app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -179,7 +183,7 @@ Fae.form.filtering = {
   _addSortingListeners: function() {
     var _this = this;
 
-    $('th[data-sort]').on('click', function() {
+    $('body').on('click', 'th[data-sort]', function() {
```

Comment:
@tshedor It looks like FCH.$body returns an empty array. Any idea why that might be happening?